devtools-ci-index.yaml

```diff
@@ -60,6 +60,7 @@
         - Katka92
         - kbsingh
         - kishansagathiya
+        - kbaig
```

Comment:
this is not correct, it should be `khrm` (your github handle)

Can you send this as a separate PR and have @pradeepto +1 the request?